src/Equations/damaged-elastic/Kernels/Time.cpp

```diff
@@ -703,3 +700,1491 @@ void seissol::kernels::Time::flopsTaylorExpansion(long long& nonZeroFlops,
 }
 
 unsigned int* seissol::kernels::Time::getDerivativesOffsets() { return m_derivativesOffsets; }
+
+void seissol::kernels::Time::calculateEps(
```

Comment:
Return a tuple and use a structured binding in the calling code to extract the tuple into its components.

Reply:
Done